File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/body/user/GetUserCurrentStatusBody.java
package com.lattis.ellipse.data.network.model.body.user;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 9/6/17.
 */


public class GetUserCurrentStatusBody {

    @SerializedName("device_model")
    private String device_model;

    @SerializedName("device_os")
    private String device_os;

    public GetUserCurrentStatusBody(String deviceModel, String deviceOS) {
        this.device_os = deviceOS;
        this.device_model = deviceModel;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/body/user/GetUserCurrentStatusBody.kt
package com.lattis.data.entity.body.user

import com.google.gson.annotations.SerializedName

/**
 * Created by ssd3 on 9/6/17.
 */
class GetUserCurrentStatusBody(
    @field:SerializedName("device_model") private val device_model: String,
    @field:SerializedName("device_os") private val device_os: String,
    @field:SerializedName("device_language") private val deviceLanguage: String
)